mage/FormShortcuts.cs

```diff
@@ -14,6 +15,12 @@ public FormShortcuts(FormMain main)
         {
             InitializeComponent();
 
+            ThemeSwitcher.ChangeTheme(Controls, this);
+            ThemeSwitcher.InjectPaintOverrides(Controls);
+
+            ThemeSwitcher.ChangeTheme(this.Controls, this);
+            ThemeSwitcher.InjectPaintOverrides(this.Controls);
+
```

Comment:
Looks like this is unnecessary

Reply:
No this is necessary in every form to do the actual theme changing

Reply:
Look at the full context, it's a duplicate